docs/docker-compose/docker-compose.sample.yml

```diff
@@ -0,0 +1,157 @@
+      - ALPINE_DATABASE_POOL_IDLE_TIMEOUT=300000
+      - ALPINE_DATABASE_POOL_MAX_LIFETIME=600000
+      #  ---- snip ----
+      # Optional HTTP Outbound Connection Timeout Settings. All values are in seconds.
+      - ALPINE_HTTP_TIMEOUT_CONNECTION=30
+      - ALPINE_HTTP_TIMEOUT_SOCKET=30
+      - ALPINE_HTTP_TIMEOUT_POOL=60
+      # Optional logging configuration
+      - LOGGING_LEVEL=INFO
+      - LOGGING_CONFIG_PATH=logback.xml
+      # Optional configuration for the OSS Index analyzer
+      - OSSINDEX_REQUEST_MAX_PURL=128
+      - OSSINDEX_RETRY_BACKOFF_MAX_ATTEMPTS=50
+      - OSSINDEX_RETRY_BACKOFF_MULTIPLIER=2
+      - OSSINDEX_RETRY_BACKOFF_MAX_DURATION=600000
+    env_file:
+      - sample.env
+    deploy:
+      resources:
+        limits:
+          memory: 8192m
+        reservations:
+          memory: 6144m
+      restart_policy:
+        condition: on-failure
+    ports:
+      - '127.0.0.1:8081:8080'
+    volumes:
+      # Optional volume mount to override default notification publisher templates
+      # - "/host/path/to/template/base/dir:/data/templates"
+      - 'dependency-track:/data'
+    restart: unless-stopped
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.dtrack-apiserver.rule=Host(`dt.example.com`) && PathPrefix(`/api`)"
+      - "traefik.http.routers.dtrack-apiserver.entrypoints=websecure"
+      - "traefik.http.routers.dtrack-apiserver.tls.certresolver=myresolver"
+
+  dtrack-frontend:
+    image: dependencytrack/frontend
+    container_name: "dtrack-frontend"
+    depends_on:
+      - dtrack-apiserver
+    environment:
+      # The base URL of the API server.
+      # NOTE:
+      #   * This URL must be reachable by the browsers of your users.
+      #   * The frontend container itself does NOT communicate with the API server directly, it just serves static files.
+      #   * When deploying to dedicated servers, please use the external IP or domain of the API server.
+      - API_BASE_URL=https://dt.example.com
+      # - "OIDC_ISSUER="
+      # - "OIDC_CLIENT_ID="
+      # - "OIDC_SCOPE="
+      # - "OIDC_FLOW="
+      # - "OIDC_LOGIN_BUTTON_TEXT="
+      # volumes:
+      # - "/host/path/to/config.json:/app/static/config.json"
+    ports:
+      - "127.0.0.1:8080:8080"
+    restart: unless-stopped
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.dtrack-frontend.rule=Host(`dt.example.com`)"
+      - "traefik.http.routers.dtrack-frontend.entrypoints=websecure"
+      - "traefik.http.routers.dtrack-frontend.tls.certresolver=myresolver"
+
+  db:
+    image: postgres:16.1-alpine
+    container_name: "db"
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=dtrack
+    env_file:
+      - sample.env
+    ports:
+      - '127.0.0.1:5432:5432'
+    volumes:
+      - ./postgres-data:/var/lib/postgresql/data
+
+
+  technolinator:
+    image: "ghcr.io/mediamarktsaturn/technolinator:fat-1.57.3"
```

Comment:
It's possible to use the default image here and add a comment describing the fat image usage?